### ISSUE TITLE
NextCloud Connection Error

### ISSUE THREAD

**Issue by MikeP1990** [ONLYOFFICE/CommunityServer#213]
Hello,

When trying to connect to NextCloud through OnlyOffice community edition, I'm receiving different errors, mostly a 503.  I'm wondering if anyone can shed some light on this?

I installed OnlyOffice using the RPM auto installer instructions found here: https://helpcenter.onlyoffice.com/server/docker/opensource/opensource-script-installation.aspx

Thanks in advance!

**Comment by ShockwaveNN** [comment]
I think you missed product, you need not OnlyOffice community edition, but ONLYOFFICE DocumentServer CommunityEdition, instruction is [here](https://helpcenter.onlyoffice.com/server/docker/document/docker-installation.aspx)

**Comment by MikeP1990** [comment]
So I only need to install the document server version and not community edition? We don’t want to use docker due to the lack of knowledge.

**Comment by ShockwaveNN** [comment]
yeah. you can install DocumentServer any way you prefer

**Comment by MikeP1990** [comment]
I was under the impression that community edition installed everything? At least that’s what it suggests in the documents?

**Comment by ShockwaveNN** [comment]
> I was under the impression that community edition installed everything

Yes it is, but I think if you want integration with Nextcloud I think you don't need ONLYOFFICE Community, whitch handle Document storage

And to be clear - do not be confuse ONLYOFFICE Community - it's a product, and ONLYOFFICE Community Edition - this is typo of installation, including ONLYOFFICE Community and ONLYOFFICE DocumentServer
